designer/server/src/common/helpers/auth/user-session.js

```diff
@@ -64,6 +65,18 @@ export async function createUserSession(request, artifacts) {
   return server.methods.session.get(user.id)
 }
 
+export function hapiScopeOptions(scope = scopes.SCOPE_READ) {
+  return {
+    auth: {
+      mode: 'try',
+      access: {
+        entity: 'user',
+        scope: [`+${scope}`]
+      }
+    }
+  }
+}
+
```

Comment:
I didn't find any other way to set options in 1 place for all routes. There is probably a better way to do this ?!